### ISSUE TITLE
SickGear <-> nzbToMedia <-> SABnzbd show name with year issue

### ISSUE THREAD

**Issue by ouija** [clinton-hall/nzbToMedia#1869]
**Describe the bug**
This may or may not be a direct result of SickGear but rather may be a configuration issue or something else, but figured I'd pose the question in case you have any insight.  _(I've also [posed this question ](SickGear/Offline#18)to the authors of SickGear as well)_

I recently switched to SickGear, and have it working beautifully with SABnzbd and nzbToMedia scripts -- However, there is one thing that I'm trying to resolve, which is minor but somewhat annoying:

I've found that completed downloads don't seem to be moving into the "Show Name" directory (defined in SickGear) when processing is completed, unless I have TV Sorting enabled in SABnzbd -- which is fine and seems to resolve this, however, this can cause isssues if there are different shows with the same name and the year is then added ti the show name.

For example, let's say a show is called "Example (2021)", and this is the folder name that SickGear created when adding the show and configured to use. When a download is completed and processed (using nzbToMedia), SABnzbd seems to be creating a folder instead called "Example 2021" (without brackets around the year) and placing the download within this folder, instead of the one defined in SickGear.

If I disable TV Sorting within SABnzbd, then this download is placed under a folder based on the downloaded file name, and not the folder defined for the show in SickGear.

It's not a major thing, because I've been renaming folders and re-mapping them in SickGear, but where it is a bit of a pain is if you are adding new shows in SickGear and if the show name has a year in it, and I figured I'd ask in case it's something as simple as an incorrect configuration problem.

Thanks!

**Technical Specs**
1. Running on QNAP TS-870 Pro
2. Python version: 3.10.0
3. Download Client: SABnzbd
4. Intended Media Management: [SickGear](https://github.com/SickGear/SickGear)

**Expected behavior**
Shows to be moved into the correct directory as defined within SickGear.

**Comment by clinton-hall** [comment]
Ok, so SABnzbd TV sorting is just using SAB rules to rename. This should be turned off.

When this is turned off, check the logs and post here so we can see what is going wrong. The SAB logs and nzbtoMedia.log should show au what is happening.

As a best guess, i would say there is a config wrong and nzbToMedia is NOT sending this to SickGear and therefore it is not being renamed.
I would check the SickGear api is correctly entered in autoProcessMedia.cfg

**Comment by ouija** [comment]
All good, it does work _properly_ if I configure it _properly_... hahaha

But I realized now why I had things configured the way I did -- Sometimes just download "one-off" episodes, and SickGear won't process them **unless they exist in the show database** _(or as alt title)_ and sometimes I just want to download one or two episode of a certain show **without** adding it to SickGear but still need it to be processed/renamed.

The way I have things configured currently do allow for this, but do cause this other issue, which I'll just have to live with for now or learn to change my stubborn ways if I want to make my life a little easier!

Thanks for your insights and for a great script!
